server/routes.go

```diff
@@ -265,6 +270,10 @@ func (s *RESTServer) addRoutes() http.Handler {
 		{"GET", "/upload/:fileid/metadata", RoleMDOnly, s.GetFileInfoHandler},
 		{"PUT", "/upload/:fileid/metadata", RoleWrite, s.SetFileInfoHandler},
 
+		// /admin/tape_use (enable, disable, get status)
+		{"GET", "/admin/use_tape", RoleWrite, s.GetTapeUseHandler},
+		{"PUT", "/admin/use_tape/:status", RoleWrite, s.SetTapeUseHandler},
```

Comment:
Should these have `RoleWrite` or `RoleAdmin`. I'm not sure, and don't have an opinion.